app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -221,13 +228,13 @@ h3
 
 .check-out-message
   background var(--app-fg)
+  border 1px solid var(--bc-dim)
```

Comment:
using some style formatting plugin?

Reply:
yes. he auto-sorts it alphabetically. it's beautiful. 😢 <-- tears of joy

Reply:
can prettier do this?

Reply:
No, [Prettier doesn't support Stylus](https://github.com/prettier/prettier/issues/2465).

Reply:
what i'm hearing is that we all want to switch to plain css 😉 

no his editor has a plugin or something like that.

Reply:
It's [Stylus Supremacy](https://thisismanta.github.io/stylus-supremacy/#).  Good news @jbibla, there's an Atom plugin!

Reply:
the best way to format stylus is to write css :trollface: